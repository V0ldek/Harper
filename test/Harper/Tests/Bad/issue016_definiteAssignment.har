main :: Integer;
main = {
  var (x :: Integer);
  return x;
};