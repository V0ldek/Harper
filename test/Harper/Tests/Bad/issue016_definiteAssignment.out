Error: variable `x` is not definitely assigned at point of use.
During evaluation of:
  x
Located at line 4 column 10

Execution terminated with an error: type error.